### DISCUSSION TITLE
One light switch start controlling all the lights in the house

### DISCUSSION THREAD

**Discussion by gunleik** [Koenkk/zigbee2mqtt#19754] [General]
I have a Xiaomi light switch in my dining room https://www.zigbee2mqtt.io/devices/WXKG15LM.html#xiaomi-wxkg15lm that became unstable. First tried to change battery on it, but that didn't help.

So I tried to re-pair it with z2m in home assistant. But now this switch have started toggling all the lights in my house... So turning on the light in the dining room will turn off the light in the kitchen, turn on the light in the bathroom and so on with all the other lights I have (so not in sync, just toggling from the last state).

I can even power off home assistant and this switch still is the master of the universe :stuck_out_tongue_winking_eye: 

Tried to delete it from z2m and re-pair it, but I guess this will not help as it seems that it controls all the lights directly without the need of z2m/home assistant.

Can anyone explain what have happened?
For now I have just replaced the switch to regain control of my house, but any tips of how to get this switch up and running normally again?

**Comment by eryq78** [comment]
I've got the same. Replacing battery didnt do anything. 
When i click a button on the switch a Blitzwolf sph-13 disables. Clicking multiple times disables the other blitzwolf smart plugs... 

Latest version of Z2Q and Home assistant.
